tasks/git.py

```diff
@@ -24,3 +23,40 @@ def tag(ctx, force=False):
         check=True,
         cwd=PROJ_ROOT,
     )
+
+
+def get_latest_release_tag():
+    gh_url = "https://api.github.com/repos/faasm/faabric/releases/latest"
+
+    curl_cmd = "curl --silent {}".format(gh_url)
+    print(curl_cmd)
+    result = run(curl_cmd, shell=True, stdout=PIPE, stderr=STDOUT)
+
+    tag = json.loads(result.stdout.decode("utf-8"))["tag_name"]
+
+    return tag
+
+
+@task
+def release_body(ctx, file_path="/tmp/release_body.md"):
+    """
+    Generate body for release with detailed changelog
+    """
+    docker_cmd = [
+        "docker run -t -v",
+        "{}/..:/app/".format(PROJ_ROOT),
```

Comment:
Note: this only works if faabric is used as a submodule in faasm. This is because, as a submodule, the `.git` folder is a softlink to `../.git/modules/faabric`.

Reply:
We shouldn't assume this, Faabric should be treated as a separate library and worked on in a separate checkout. Editing Faabric as a submodule of Faasm should be the exception not the rule.